app/assets/javascripts/models/beta/history.js

```diff
@@ -0,0 +1,9 @@
+export const getDate = (key, changes) => {
+  if (key.endsWith("_at")) {
+    return [
+      changes[key][0] ? I18n.l("date.formats.long", changes[key][0]) : null,
```

Comment:
You can create a variable with `changes[key][0]` and `changes[key][1]` with meaningful names.
I believe `changes` is an `Object` where each value is an `Array` with the previous value and the next value so:

```js
export const getDate = (key, changes) => {
  const [previousValue, nextValue] = changes[key];
  
  if (key.endsWith("_at")) {  
    return [
      previousValue ? I18n.l("date.formats.long", previousValue) : null,
      previousValue ? I18n.l("date.formats.long", nextValue) : null
    ];
  }

  return [previousValue, nextValue];
};
```